src/utils/artifactUtils.tsx

```diff
@@ -300,6 +301,21 @@ export const getThreadID = (args: {
     return null;
 };
 
+export const getArtifactProduct = (artifact: Artifact): string | null => {
+    if (isArtifactMBS(artifact) || isArtifactRPM(artifact)) {
+        const { gate_tag_name } = artifact.payload;
+        if (isEmpty(gate_tag_name)) {
+            return null;
+        }
+        const product = gate_tag_name.match(/^.*(rhel-\d\d?)\.$/);
```

Comment:
This only works for RHEL. Do we want to support Fedora and CentOS, as well?

Reply:
Gating based on brew tags is available only in RHEL.
Cenots/Fedora doesn't have this.

Reply:
Can you please mention that in a comment?